src/Tasks/Resources/xlf/Strings.zh-Hant.xlf

```diff
@@ -3110,8 +3110,8 @@
       </trans-unit>
       <trans-unit id="WriteLinesToFile.ErrorReadingFile">
         <source>MSB3492: Could not read existing file "{0}" to determine whether its contents are up to date. Overwriting it.</source>
-        <target state="translated">>MSB3492: 無法讀取現有的檔案 "{0}"，所以無法判斷其是否包含最新的內容。將予覆寫。</target>
-        <note>{StrBegin="MSB3491: "}</note>
+        <target state="needs-review-translation">&gt;MSB3492: 無法讀取現有的檔案 "{0}"，所以無法判斷其是否包含最新的內容。將予覆寫。</target>
```

Comment:
This looks fishy

Reply:
@v-zbsail, @natemcmaster, is it okay to just remove the extra `&gt`?

Reply:
also @tmeschter ☝

Reply:
Talked to Tom offline. Removing it and leaving it marked as `needs-review-translation`, so the loc team can take a look in their next pass.